pages/gacha.html

```diff
@@ -145,14 +145,17 @@ <h1 id="title-header">The Regis Altare Birthday Project 2023</h1>
           />
           <label for="gacha-grid"> Card Grid</label>
         </form>
-        <button id="gacha-button" type="submit">Open Card Pack</button>
         <div id="gacha-notes">
+          <p class="visually-hidden">
+            Card Grid is the recommended view for screenreader users.
+          </p>
```

Comment:
In pile view, the screenreader loses its position if you flip cards "out of order":
 - you're in grid view and go down to flip the 7th card, then you switch back to pile view. I don't expect screenreaders to use the site in this way since the view is purely visual, so I'm not concerned about solving this
 - you can reach the lower cards with screenreader and flip them), which disorients screenreader users. The solution would be to programmatically toggle tabbing and aria-hidden as cards are flipped so only the frontmost card is accessible, but that would be a nontrivial effort. So I compromised with this note for screenreader uses :')